protostar-rust/tests/data/deploy_test/tests/test_deploy.cairo

```diff
@@ -0,0 +1,19 @@
+use result::ResultTrait;
+use protostar_print::PrintTrait;
+use array::ArrayTrait;
+use cheatcodes::PreparedContract;
+
+#[test]
+fn test_deploy_simple() {
+    let calldata = ArrayTrait::<felt252>::new();
+    let class_hash = declare('deploy_test').expect('Could not declare');
+
+    let contract_address = deploy(
+        PreparedContract {
+            class_hash,
```

Comment:
What will happen if non-existing class_hash is passed?

Reply:
I guess panic, but this can be tested